dropwizard/service/src/test/resources/polaris-server-integrationtest.yml

```diff
@@ -162,3 +162,8 @@ tokenBucketFactory:
   type: default
   requestsPerSecond: 9999
   windowSeconds: 10
+
+# Fake GCP credentials for tests that do not need access to storage
+gcp_credentials:
+  access_token: "abc"
+  expires_in: 12345
```

Comment:
Is this related to the PR?

Reply:
Actually, it turned out fake GCP credentials are still required for test, but the error shows up only in GH CI, not in local runs.